### PR TITLE
[front] fix: more involving tutorial messages

### PR DESCRIPTION
**Pull request by GresilleSiffle**
**related to** #1099

This small PR adds few words in the tutorial to make it clear that:
- a **comparison is actually a contribution** to the project
- Tournesol uses comparisons **to make recommendations**
- going through the tutorial **helps** the Tournesol project

We might want one day to review the tutorial texts in details, in the meantime this is my subjective attempt to make the tutorial more clear without increasing its size and without adding too much text.

Hope it helps : )

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -574,14 +574,14 @@
       "tutorial": {
         "title1": "Welcome to Tournesol 🌻",
         "message1": {
-          "p10": "This tutorial will guide you step by step through a series of four comparisons.",
+          "p10": "This tutorial will guide you, step by step, through a series of four comparisons to enrich the project.",
```
Comment (lfaucon) [inline frontend/public/locales/en/translation.json:577]: I don't like "enrich" here. I'd propose "contribute to"

```suggestion
          "p10": "This tutorial will guide you, step by step, through a series of four comparisons to help you contribute to the Tournesol project",
```

File: frontend/public/locales/en/translation.json [hunk 2]
```diff
@@ -574,14 +574,14 @@
...
           "p20": "To compare two videos you watched, move the main handle and save when your choice is made.",
           "p30": "The more your opinion is clear-cut on a criterion, the more the handle should be close to the slider extremity.",
           "p40": "If you find the videos similar, the handle should remain close to the center."
         },
         "title2": "Bravo ! 🥳",
         "message2": {
-          "p10": "You have made your first comparison",
+          "p10": "You have made your first comparison. Each of them allows Tournesol to fine-tune its recommendations.",
```
Comment (lfaucon) [inline frontend/public/locales/en/translation.json:584]:
```suggestion
          "p10": "You have made your first comparison. Each comparison helps Tournesol to improve its video recommendations.",
```

**Review by lfaucon — APPROVED** [review]
Left suggestions, but I don't have a very strong opinion on it.

File: frontend/public/locales/en/translation.json (hunk 1 above)
Comment (lfaucon) [reply frontend/public/locales/en/translation.json:577]: Would recommend updating the French too

File: frontend/public/locales/en/translation.json (hunk 2 above)
Comment (GresilleSiffle) [reply frontend/public/locales/en/translation.json]: Fixed by https://github.com/tournesol-app/tournesol/pull/1110/commits/d30471e2fc49580aab2a884f98a82ec0da96f69c.

File: frontend/public/locales/en/translation.json (hunk 1 above)
Comment (GresilleSiffle) [reply frontend/public/locales/en/translation.json]: Fixed by https://github.com/tournesol-app/tournesol/pull/1110/commits/d30471e2fc49580aab2a884f98a82ec0da96f69c.

**Pull request merged**